sections/transactions.js

```diff
@@ -80,6 +80,75 @@ export default [
       });
     `
   },
+  {
```

Comment:
Was this part supposed to be included in this PR?..

Reply:
No, don't know who it got there :(